src/app/lib/utilities/formatDuration/index.js

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable prefer-template */
-
 const lpad = s => s.toString().padStart(2, '0');
-
 // the duration argument should be a moment duration!
-const formatDuration = (duration, separator = ':') => {
+const formatDuration = (duration, separator = ':', articlesPage = false) => {
```

Comment:
You might want to change the `params` type to an `object` instead of having multiple optional parameters.

Reply:
Refactored as suggested

Reply:
This introduced a lot of complexity to the PR than need be so had to revert back

Reply:
Looks good now, thanks for updating.
https://github.com/bbc/simorgh/blob/9dd2671597a34621bb385850e3e859342b340b00/src/app/lib/utilities/formatDuration/index.js#L4